<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1296</width>
    <height>704</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mon zoo</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_zooName">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>30</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoo Ynov</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_updateZooName">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>40</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Modifier</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_zooName">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>60</y>
      <width>113</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_validateZooName">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>40</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Valider</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_budgetTitle">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>140</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Budget :</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_budget">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>150</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>150</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>€</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_earn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>150</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>93</width>
      <height>0</height>
     </size>
    </property>
    <property name="text">
     <string>+50 €</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btn_spend">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>150</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-30 €</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStockViande">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>190</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Stock de viande :</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelQuantityStockMeat">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>200</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSymbolKg">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>200</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>kg</string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddMeat">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+50 kg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRemoveMeat">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>200</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>- 30 kg</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelStockSeed">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>240</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Stock de graine : </string>
    </property>
   </widget>
   <widget class="QLabel" name="labelQuantityStockSeed">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>250</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="labelSymbolKg_2">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>250</y>
      <width>31</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>kg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddSeed">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>+ 50 kg</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonRemoveSeed">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>250</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>- 30 kg</string>
    </property>
   </widget>
   <widget class="QListView" name="listViewLogs">
    <property name="geometry">
     <rect>
      <x>995</x>
      <y>0</y>
      <width>301</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableViewHabitat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>290</y>
      <width>721</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAddHabitat">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>590</y>
      <width>161</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Ajout d'un habitat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSellHabitat">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>590</y>
      <width>161</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Vendre un habitat</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonDestroyHabitat">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>590</y>
      <width>161</width>
      <height>34</height>
     </rect>
    </property>
    <property name="text">
     <string>Détruire un habitat</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1296</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
